main.go

```diff
@@ -51,6 +64,66 @@ func (c *runCmd) Run() error {
 	return nil
 }
 
+func (c *fmtCmd) Run() error {
+	if len(c.Files) == 0 {
+		return format(os.Stdin, os.Stdout, c.Check)
```

Comment:
You should probably return an error here if `c.Write` is true - can't write to files that are not listed